tests/test_coords.py

```diff
@@ -8,7 +9,7 @@
 
 
 @given(tuples(floats(), floats(), floats()))
-def test_valid_conversion(euler_angles):
+def test_valid_conversion(euler_angles: Tuple[float, float, float]) -> None:
```

Comment:
Having to type all test cases as returning `None`, that's far too messy. Is there any way to not need this?

Reply:
We could add an exception for the `tests` directory in `mypy` I guess?

It's what we do in `j5` and it has worked fine there for months. See https://github.com/j5api/j5/pull/210

Reply:
I'm not talking about whether it works, I'm just saying it shouldn't be necessary. It leads to a large amount of noise in the code, and adds more for people to think about.

Does removing them start raising type errors? Explicit `None` return types feels unnecessary

Reply:
If we want to enforce typed function definitions, which I really think we should because I have no desire to do this all again, we need the following mypy options:

```
disallow_untyped_defs = True
disallow_incomplete_defs = True
```

Removing None from these tests would violate both rules, and the tests aggressively fail with hundreds of errors. 

We should not remove these rules, unless you think that it is acceptable to have missing types. (imo there should be at least an Any if the type is unknown)

Reply:
That's a shame.

I'd definitely prefer to have everything checked rather than none of it checked.